manual/content/point-data/index.md

```diff
@@ -0,0 +1,10 @@
+# Point Data
+
```

Comment:
The main page should probably have some general information about the point cloud support, now that I think about it